lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -48,6 +48,14 @@ class OpenResponseMetadataSerializer(serializers.Serializer):
     name = serializers.CharField(source='display_name')
     prompts = serializers.ListField()
     type = serializers.SerializerMethodField()
+    textResponseConfig = serializers.SerializerMethodField()
+    fileUploadResponseConfig = serializers.SerializerMethodField()
+
+    def get_textResponseConfig(self, instance):
+        return instance.text_response or 'none'
+
+    def get_fileUploadResponseConfig(self, instance):
+        return instance.file_upload_response or 'none'
```

Comment:
There's really no reason to make this a `SerializerMethodField` and should be a `ChoiceField` instead.

```suggestion
    textResponseConfig = serializers.ChoiceField(choices=['optional', 'required', 'none'], 'source='text_response')
    fileUploadResponseConfig = serializers.ChoiceField(choices=['optional', 'required', 'none'], source='file_upload_response')
```